src/fluree/db/json_ld/shacl.cljc

```diff
@@ -436,100 +458,100 @@
   "Builds map out of values from a SHACL propertyShape (target of sh:property)"
   [property-flakes]
   (reduce
-    (fn [acc property-flake]
-      (let [o (flake/o property-flake)]
-        (condp = (flake/p property-flake)
-          const/$sh:path
-          (update acc :path (fnil conj []) o)
+   (fn [acc property-flake]
+     (let [o (flake/o property-flake)]
+       (condp = (flake/p property-flake)
+         const/$sh:path
+         (update acc :path (fnil conj []) o)
 
-          ;; The datatype of all value nodes (e.g., xsd:integer).
-          ;; A shape has at most one value for sh:datatype.
-          const/$sh:datatype
-          (assoc acc :datatype o)
+         ;; The datatype of all value nodes (e.g., xsd:integer).
+         ;; A shape has at most one value for sh:datatype.
+         const/$sh:datatype
+         (assoc acc :datatype o)
 
-          const/$sh:minCount
-          (cond-> (assoc acc :min-count o)
-            (>= o 1) (assoc :required? true)) ; min-count >= 1 means property is required
+         const/$sh:minCount
+         (cond-> (assoc acc :min-count o)
+           (>= o 1) (assoc :required? true)) ; min-count >= 1 means property is required
 
-          const/$sh:maxCount
-          (assoc acc :max-count o)
+         const/$sh:maxCount
+         (assoc acc :max-count o)
 
-          ;; values of sh:nodeKind in a shape are one of the following six instances of the
-          ;; class sh:NodeKind: sh:BlankNode, sh:IRI, sh:Literal sh:BlankNodeOrIRI,
-          ;; sh:BlankNodeOrLiteral and sh:IRIOrLiteral.
-          ;; A shape has at most one value for sh:nodeKind.
-          const/$sh:nodeKind
-          (assoc acc :node-kind o)
+         ;; values of sh:nodeKind in a shape are one of the following six instances of the
+         ;; class sh:NodeKind: sh:BlankNode, sh:IRI, sh:Literal sh:BlankNodeOrIRI,
+         ;; sh:BlankNodeOrLiteral and sh:IRIOrLiteral.
+         ;; A shape has at most one value for sh:nodeKind.
+         const/$sh:nodeKind
+         (assoc acc :node-kind o)
 
-          ;; Note that multiple values for sh:class are interpreted as a conjunction,
-          ;; i.e. the values need to be SHACL instances of all of them.
-          const/$sh:class
-          (update acc :class (fnil conj []) o)
+         ;; Note that multiple values for sh:class are interpreted as a conjunction,
+         ;; i.e. the values need to be SHACL instances of all of them.
+         const/$sh:class
+         (update acc :class (fnil conj []) o)
```

Comment:
Since we convert this to a set when we use it, should we just start with `(fnil conj #{})` here?